.github/workflows/test.yml

```diff
@@ -0,0 +1,78 @@
+name: Test
+on: [push]
+jobs:
+  test-linux:
+    name: Shorter linux tests
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Setup
+      uses: dpryan79/github-actions/bioconda_utils_setup_conda/@master
```

Comment:
should this be migrated to bioconda?

Reply:
Yes, if people agree that we should continue on with github actions then I'll create the repos.

Reply:
We now have a bioconda-actions repository for things like that.